test/prepareTest.ts

```diff
@@ -2,14 +2,20 @@ import ENV from "../src/utils/env.js"
 import mongoose from "mongoose"
 import backend from "../src/app.js"
 
-mongoose.connect(ENV.MONGOURI, {
-	ignoreUndefined: true,
-	loggerLevel: "info"
-}) // connect to db before loading other stuff
+jest.setTimeout(10000)
+beforeAll(async () => {
+	await mongoose.connect(ENV.MONGOURI, {
+		ignoreUndefined: true,
+		loggerLevel: "info"
+	})
 
-export default backend
+	await backend.listen(0);
```

Comment:
This doesn't need to be there, because `fastify.inject` works even if the backend is not listening
```suggestion
```

Reply:
Does the WebSocket server work without listening on a port? Does fastify.inject handle upgrade requests in a way that new WebSocket(address) can be redirected to? If not then the backend will have to listen on a port.

Reply:
Hmm, not sure about the WebSocket server. The thing is, fastify.inject is supposed to simulate requests only - run handlers without actually listening on any port. I guess it won't work, so you are probably right. Will need to test it when the WS tests are added.